yarn-project/simulator/src/public/execution.ts

```diff
@@ -86,8 +86,6 @@ export interface PublicExecutionResult {
    */
   allUnencryptedLogs: UnencryptedFunctionL2Logs;
 
-  // TODO(dbanks12): add contract instance read requests
-
```

Comment:
GETCONTRACTINSTANCE is already in the hints, and doesn't need a dedicated entry here. It will ultimately just emit a nullifier.